src/__tests__/server.test.ts

```diff
@@ -74,3 +76,40 @@ test('stop twice does nothing on second attempt', async () => {
 
   fail('expected a connection error as server should be closed');
 });
+
+const getPort = async (options?: Options) => {
+  const { schema, ctxFactory } = testGraphql(options);
+  ctxFactory.config.port = await availablePort();
+
+  await startServer(schema, ctxFactory);
+
+  const address = getAddress();
+  await healthcheckOk;
+  return address.port;
+};
+
+const ropstenOptions: Options = {
+  jsonrpc: 'https://ropsten.infura.io',
+};
+
+test('JSON RPC endpoint configuration works correctly', async () => {
+  let query;
+
+  try {
+    query = await axios.post(`http://localhost:${await getPort()}/graphql`, { query: '{ block(number: 1) { hash } }' });
+  } catch (err) {
+    return;
```

Comment:
If the HTTP request raises an exception, we want the test to fail. This swallows the exception and ends the test, marking it as passed.